packages/account/src/Components/form-body/__tests__/form-body.spec.tsx

```diff
@@ -16,18 +16,20 @@ describe('<FormBody />', () => {
                 <div>Test children</div>
             </FormBody>
         );
-        expect(container.querySelector('account__scrollbars_container--grid-layout')).toBeInTheDocument;
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
```

Comment:
@yauheni-kryzhyk-deriv please consider finding elements by data-testids, or disabling these eslint rules in config instead of explicitly disabling separate lines 🙂

Reply:
ok. will rewrite tests)